audformat/core/table.py

```diff
@@ -137,8 +137,49 @@ def __setitem__(self, column_id: str, column: Column) -> Column:
             BadIdError: if a column with a ``scheme_id`` or ``rater_id`` is
                 added that does not exist
             ValueError: if column ID is not different from level names
+            ValueError: if the column is linked to a scheme
+                that is using labels from a misc table,
+                but the misc table the column is assigned to
+                is already used by the same or another scheme
 
         """
+
+        if (
+                column.scheme_id is not None
+                and self.db is not None
+                and column.scheme_id in self.db.schemes
+        ):
+
+            # check if scheme uses
+            # labels from a table
+            labels = self.db.schemes[column.scheme_id].labels
+            if isinstance(labels, str):
+
+                # check if scheme uses
+                # labels from this table
+                if self._id == labels:
```

Comment:
Please also change the code here to make use of `scheme.uses_table`

Reply:
done